components/Button.js

```diff
@@ -29,23 +30,44 @@ const Button = (props) => {
     children,
     className = "",
     loading = false,
+    tooltip = null,
     ...otherProps
   } = props;
 
   const { theme } = useTheme();
 
+  let tooltipId = null;
+
+  if (tooltip) {
+    tooltipId = `${label.replace(" ", "")}-toolTip`;
+  }
+
   return (
-    <button
-      className={`${styles[buttonType]} ${styles[theme]} ${className}`}
-      onClick={onClick}
-      aria-label={label}
-      disabled={disabled || loading}
-      {...otherProps}
-    >
-      <Loading type={"button"} loading={loading}>
-        {children || label}
-      </Loading>
-    </button>
+    <>
+      {tooltip && (
+        <ReactTooltip
+          id={tooltipId}
+          effect={"solid"}
+          place={"top"}
+          delayShow={200}
+        >
+          <p>{tooltip}</p>
```

Comment:
At least from the use in this PR, it seems like the tooltip value is always the same as the `label` prop. What do you think about having a prop called `showTooltip` (or something along those lines) that renders the label value as tooltip? Or is there a case where those might be different?

Reply:
My thinking was any other button that isn't an icon would want a different value than the label but like you said, there's not currently use case for it. I can default to the label for now and update this logic if we find a different use case later.